hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieHiveUtil.java

```diff
@@ -71,18 +71,13 @@ public static Path getNthParent(Path path, int n) {
   public static List<String> getIncrementalTableNames(JobContext job) {
     Map<String, String> tablesModeMap = job.getConfiguration()
         .getValByRegex(HOODIE_CONSUME_MODE_PATTERN_STRING.pattern());
-    List<String> result = tablesModeMap.entrySet().stream().map(s -> {
+    return tablesModeMap.entrySet().stream().map(s -> {
```

Comment:
would it return null? we need avoid return null.

Reply:
good catch - fixed it